common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -1022,4 +1022,12 @@ object Utils extends Logging {
       parentFile.lastIndexOf("/"),
       parentFile.length) + "/" + fileInfo.getFile.getName
   }
+
+  def parseMetricLabels(label: String): Tuple2[String, String] = {
+    val labelPart = label.split(":")
+    if (labelPart.size != 2) {
+      throw new IllegalArgumentException(s"Illegal metric extra labels: $label")
+    }
+    labelPart(0) -> labelPart(1)
```

Comment:
Given that the example contains a space before `,`, the result should do trim here.
```
Allowed pattern is: `<label1_key>:<label1_value> [,<label2_key>:<label2_value> ]*`
```

And, why use `k:v`? the Prometheus metric format use `k=v`, personally, `=` looks more straightforward.

Reply:
space should be removed in pattern. so this don't add trim

Reply:
but we if the user sets `env=prod , type= flink`?

Reply:
i know your meaning that user may add space. it's better to trim the extra space